src/helm/benchmark/presentation/run_specs_chat_gpt.conf

```diff
@@ -194,16 +198,18 @@ entries: [
   # For some prompts, ChatGPT responds "It is not appropriate to spread false or misleading
   # information about individuals, especially in a derogatory or defamation".
   # This is because ChatGPT using the Moderation API to warn or block unsafe content.
-  {description: "real_toxicity_prompts:model=openai/chat-gpt", priority: 2}
+  # Skip for now since it's more expensive to run (num_outputs=5)
+  # {description: "real_toxicity_prompts:model=openai/chat-gpt", priority: 2}
```

Comment:
can we run this with num_outputs = 1, might be interesting to see given a lot of attention has gone to chatgpt's moderation